cmd/main.go

```diff
@@ -36,7 +41,23 @@ var (
 )
 
 var saramaLogger = log.New(io.Discard, "[Sarama] ", log.LstdFlags)
+
+func InitTracer() *sdktrace.TracerProvider {
+	exporter, err := stdout.New(stdout.WithPrettyPrint())
+	if err != nil {
+		log.Fatal(err)
+	}
+	tp := sdktrace.NewTracerProvider(
+		sdktrace.WithSampler(sdktrace.AlwaysSample()),
+		sdktrace.WithBatcher(exporter),
```

Comment:
Yeah, it will come from the env vars by default and I think thats quite sensible, I just wanted to have a conversation as about if we would rather be explicit.

Reply:
you mean add the [link](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/jaeger#environment-variables) to the readme ?

Reply:
No I meant something more like https://github.com/open-telemetry/opentelemetry-go/blob/main/example/jaeger/main.go#L54

Reply:
oh get it. why not using the default env ? Should i pass an already defined parameter service name ?